solutions/usaco-1060.mdx

```diff
@@ -53,5 +53,77 @@ int main(){
 ```
 
 </CPPSection>
+	
+<JavaSection>
+	
+```java
+import java.util.*;
+import java.io.*;
+	
+class Main {
+	public static void main(String[] args) {
+		Scanner sc = new Scanner(System.in);
+		int n = sc.nextInt();
+		int[] petals = new int[n];
+		for (int i = 0; i < n; i++) {
+			/* 
+			 * storing each petal count in an array
+			 * so we can keep track of which flower
+			 * has how many petals based on the 
+			 * index it is stored in
+			 */
+			petals[i]=sc.nextInt();
+		}
+		int counter = 0;
+		/* 
+		 * creating 2 nested arrays to brute force every 
```

Comment:
arrays??

Reply:
actually maybe just replace this entire comment w/ `// brute force every possible subsequence of flowers`

Reply:
bruh i was doing math hw while doing the editorial 
ill fix it m bad

Reply:
> actually maybe just replace this entire comment w/ `// brute force every possible subsequence of flowers`

`// brute force every possible subarray of flowers`

Reply:
done